projects/plugins/jetpack/_inc/client/discussion/subscriptions.jsx

```diff
@@ -134,6 +139,17 @@ class SubscriptionsComponent extends React.Component {
 									'jetpack'
 								) }
 							/>
+							<ToggleControl
```

Comment:
I think it may be best to hide this with the feature flag. Otherwise folks will toggle that option, expect something to happen, and reach out to support when nothing happens.

You can check #31479 for an example of a PR that adds the necessary tooling to pull the feature flag from PHP into the dashboard here.

Reply:
[Done!](https://github.com/Automattic/jetpack/pull/31393/commits/e5134959dabfce6e9e36bb6893b8e1c4c1feac24)